daemon/src/wallet.rs

```diff
@@ -115,7 +115,7 @@ impl Actor {
 
     fn sync_internal(&mut self) -> Result<WalletInfo> {
         self.wallet
-            .sync(NoopProgress, None)
+            .sync(NoopProgress, Some(1000))
```

Comment:
would you mind adding a comment why this magic number got introduced next to it? Would be good for maintainability 👍 

(also: typo in commit desc: *upstream*)

couple of questions that the commit description could resolve:
- would it be enough for long-running umbrel nodes in production? 
- should this be reverted if upstream fixes the problem?